### PR TITLE
Adaptive blocks

### PR DESCRIPTION
**Pull request by olexandrd**

### CODE REVIEW

File: src/css/base.css [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
   --tablet-container-max-width: 704px;
   --desktop-container-max-width: 1088px;
+  --container-max-width: 335px;
```
Comment (AntonChubarov) [inline src/css/base.css:20]: As paddings is 20px, should be 375

File: src/css/base.css [hunk 2]
```diff
@@ -17,6 +17,7 @@
 
   --tablet-container-max-width: 704px;
```
Comment (AntonChubarov) [inline src/css/base.css:18]: As padding 32px, should be 768

File: src/css/base.css [hunk 3]
```diff
@@ -17,6 +17,7 @@
 
   --tablet-container-max-width: 704px;
   --desktop-container-max-width: 1088px;
```
Comment (AntonChubarov) [inline src/css/base.css:19]: As padding is 64px, should be 1216

**Review by AntonChubarov — CHANGES_REQUESTED** [review]

File: src/css/base.css (hunk 1 above)
Comment (Sysqwerty) [inline src/css/base.css:20]: --tablet-container-max-width: 768px;
  --desktop-container-max-width: 1088px;
  --container-max-width: 375px;

Comment (Sysqwerty) [reply src/css/base.css:20]: --tablet-container-max-width: 768px;
--desktop-container-max-width: 1088px;
--container-max-width: 375px;

I think that should to fit the Figma.

File: src/css/base.css (hunk 3 above)
Comment (Sysqwerty) [reply src/css/base.css:19]: @AntonChubarov this is only for 2 secctions, all other should be 1088.
So it's better to leave as is the default. And for 2 srctions (that are wider) need to override that value to 1216

Comment (AntonChubarov) [reply src/css/base.css:19]: Ok, than need to be overrided

**Review by kazamov — APPROVED** [review]

**Pull request merged**